recipes/gwcosmo/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "gwcosmo" %}
+{% set version = "2.1.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gwcosmo-{{ version }}.tar.gz
+  sha256: d8a193116b148937ed633d2fc8dd765448375580dffe700a38b77c089b3cb144
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.0
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.8
+    - astropy
+    - bilby
+    - h5py
+    - healpy
+    - lalsuite
+    - ligo-gracedb
+    - ligo-segments
+    - ligo.skymap <=1.0.7
+    - lscsoft-glue
+    - matplotlib-base >=2.0
+    - numpy <=1.23.5,>=1.19
+    - pandas
+    - progressbar
+    - scipy
+    - seaborn
+    - tqdm
+
+channel_targets:
+  - conda-forge gwcosmo_dev
```

Comment:
This should be in a file named `conda_build_config.yaml`.

Reply:
Thanks! I've moved it to that file.